### ISSUE TITLE
Unable to start Shairport-sync

### ISSUE THREAD

**Issue by mynameisdaniel32** [mikebrady/shairport-sync#882]
I've had some issues with Shairport-Sync today - it had been running it fine, but decided seems updating things on my pi has broken it. The pi needed a bit of a cleanup anyway, so I downloaded the latest raspbian (Buster), and went about installing 3.3.1 from the Sid repo on a completely fresh system and still came across the same error.

I haven't changed anything in the config file yet, just installed it and tried to start it up as you can see below.

Unsure where to start diagnosing this, before I wiped the pi I was also seeing the same error on 3.2.2. I also have another pi which I updated to Buster (from Stretch), without updating shairport-sync and that's fine. I'm wondering if a rpi firmware update might have broken something? I'm on a pi 3B.

```
pi@MusicPi:~ $ sudo systemctl start shairport-sync
Job for shairport-sync.service failed because the service did not take the steps required by its unit configuration.
See "systemctl status shairport-sync.service" and "journalctl -xe" for details.
pi@MusicPi:~ $ sudo systemctl status shairport-sync
● shairport-sync.service - ShairportSync AirTunes receiver
   Loaded: loaded (/lib/systemd/system/shairport-sync.service; enabled; vendor preset: enabled)
   Active: failed (Result: protocol) since Tue 2019-07-23 00:14:33 ACST; 5s ago
     Docs: man:shairport-sync(7)
           file:///usr/share/doc/shairport-sync/README.md.gz
           https://github.com/mikebrady/shairport-sync
  Process: 4124 ExecStart=/usr/bin/shairport-sync --daemon $DAEMON_ARGS (code=exited, status=0/SUCCESS)

Jul 23 00:14:33 MusicPi systemd[1]: shairport-sync.service: Service RestartSec=100ms expired, scheduling restart.
Jul 23 00:14:33 MusicPi systemd[1]: shairport-sync.service: Scheduled restart job, restart counter is at 5.
Jul 23 00:14:33 MusicPi systemd[1]: Stopped ShairportSync AirTunes receiver.
Jul 23 00:14:33 MusicPi systemd[1]: shairport-sync.service: Start request repeated too quickly.
Jul 23 00:14:33 MusicPi systemd[1]: shairport-sync.service: Failed with result 'protocol'.
Jul 23 00:14:33 MusicPi systemd[1]: Failed to start ShairportSync AirTunes receiver.

```

**Comment by mikebrady** [comment]
Thanks for the post. The Sid repos are still unstable, and a bug has been discovered and fixed in the latest `development` branch that could potentially be causing problems the Sid repository.

The best way to get the latest Shairport Sync would be to build it from here – switch to the `development` branch to get the latest.

**Comment by mynameisdaniel32** [comment]
Thanks - building from dev has fixed it - just noticed 3.3.2 was released, might give that a go when I get a minute.
